config-default.yml

```diff
@@ -521,6 +521,7 @@ voice_gate:
     minimum_days_verified: 3  # How many days the user must have been verified for
     minimum_messages: 50  # How many messages a user must have to be eligible for voice
     bot_message_delete_delay: 10  # Seconds before deleting bot's response in Voice Gate
+    minimum_activity_blocks: 3 # Number of 10 minute blocks during which a user must have been active
```

Comment:
Although it's not Python code the rest of the config file still uses two spaces before the hash.